operations/rates.md

```diff
@@ -135,6 +134,7 @@ Extent of data to be returned.
 | `ExternalNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the external name of the rate. |
 | `Description` | [Localized text](_objects.md#localized-text) | optional | All translations of the description of the rate. |
 | `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of the rate from external system. |
+| ~~`BaseRateId`~~ | ~~string~~ | ~~optional~~ | ~~Unique identifier of the base `Rate`.~~ **Deprecated!** Use DependentRatePricing.|
```

Comment:
This reminded me that BaseRateId should not be deprecated yet, because the replacement fields are not yet in documentation. The related PR: https://github.com/MewsSystems/mews/pull/62499